__tests__/unit/graphql/resolvers/distance/mutations.test.js

```diff
@@ -0,0 +1,82 @@
+// import jest from "jest";
+import * as mockingoose from "mockingoose";
+import { Distance, Category, Score, Series } from "$/db/index.js";
+import mutations from "$/graphql/resolvers/distance/mutations.js";
+import transform_by_JSON from "$/utils/transform_by_JSON.js";
+
+let Distance_data = {
+    "_id": "60bb8025b5c13d62808bb61a",
+    "competition_id": "60aff2a49d916cd1cec8629a",
+    "category_id": "60b94ff8446e241ac47ecdfe",
+    "name": "90m",
+    "series_type": 6,
+    "number_of_arrows": 6,
+}
+
+let Distances_data = [
+    {
+        "_id": "60bb8025b5c13d62808bb61a",
+        "competition_id": "60aff2a49d916cd1cec8629a",
+        "category_id": "60b94ff8446e241ac47ecdfe",
+        "name": "90m",
+        "series_type": 6,
+        "number_of_arrows": 6,
+    },
+    {
+        "_id": "60bb8030b5c13d62808bb61d",
+        "competition_id": "60aff2a49d916cd1cec8629a",
+        "category_id": "60b94ff8446e241ac47ecdfe",
+        "name": "70m",
+        "series_type": 6,
+        "number_of_arrows": 3,
+    }
+]
+
+describe("Test mutation resolvers for distance", () => {
+    beforeAll(async () => {
+        mockingoose(Distance)
+            .toReturn(Distance_data, "create")
+            .toReturn(Distance_data, "findOne")
+            .toReturn(Distance_data, "findOneAndDelete")
+            .toReturn(Distance_data, "findOneAndUpdate");
+        mockingoose(Category);
+        mockingoose(Score);
+        mockingoose(Series);
+    });
+
+    afterAll(() => {
+        jest.clearAllMocks();
+        mockingoose.resetAll();
+    });
+
+    describe("addDistance",()=>{
+        test("should return created document.",async ()=>{
+            let result = transform_by_JSON(await mutations.addDistance(null,Distance_data,null,null));
+            
+            expect(result) .toMatchObject(Distance_data);
```

Comment:
Remove space before dot in each test. Is Prettier working?

Reply:
Prettier works only when I use vs code to commit. 
When I use GitKraken, the scripts are not executed.

Reply:
So don't use GitKraken or make it use Prettier.